### DISCUSSION TITLE
no completions were found

### DISCUSSION THREAD

**Discussion by FanJiGang** [community/community#14684] [Copilot Conversations]
No matter what I type, it says "no completions were found"

![image](https://user-images.githubusercontent.com/45392954/162686647-5f0cc0c3-83cf-40d3-93e9-fadda0bc6eec.png)

**Comment by bmuskalla** [comment]
Thanks for reporting this. Could you please provide IntelliJ version/edition, Copilot plugin version and what kind of file you were working with? Best include a sample file that shows no suggestions.

**Reply by jiandan15** [reply]
> 感谢您报告此事。您能否提供 IntelliJ 版本/版本、Copilot 插件版本以及您使用的文件类型？最好包含一个不显示任何建议的示例文件。
me too,
![1649836278(1)](https://user-images.githubusercontent.com/49307773/163127338-4f7a5b5a-68a6-411c-8b69-257acd69ed0d.png)

IntelliJ IDEA 2021.1 x64,  github-copilot-intellij-1.1.16.1219
